src/modules/colorPicker/ColorPickerUI/ZoomWindow.xaml.cs

```diff
@@ -10,50 +10,16 @@ namespace ColorPicker
     /// <summary>
     /// Interaction logic for ZoomWindow.xaml
     /// </summary>
-    public partial class ZoomWindow : Window, INotifyPropertyChanged
+    public partial class ZoomWindow : Window
     {
-        private double _left;
-        private double _top;
-
         public ZoomWindow()
         {
             InitializeComponent();
             DataContext = this;
-        }
-
-        public double DesiredLeft
-        {
-            get
-            {
-                return _left;
-            }
-
-            set
-            {
-                _left = value;
-                NotifyPropertyChanged(nameof(DesiredLeft));
-            }
-        }
 
-        public double DesiredTop
-        {
-            get
-            {
-                return _top;
-            }
-
-            set
-            {
-                _top = value;
-                NotifyPropertyChanged(nameof(DesiredTop));
-            }
-        }
-
-        public event PropertyChangedEventHandler PropertyChanged;
-
-        private void NotifyPropertyChanged(string propertyName)
-        {
-            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+            // must be large enough to fit max zoom
+            Width = 500;
+            Height = 500;
```

Comment:
Why are you setting height here and not in the xaml when it is hardcoded? 
Also did you check this on 2 monitors with different DPI? I am afraid it might not work correctly

Reply:
> Why are you setting height here and not in the xaml when it is hardcoded?

In the future, I want the width/height to be exactly as large as the max zoom window, so that it's simpler to change window styling without worrying about changing the code, but I couldn't figure out a way to do that.
I thought setting the width/height on window creation would be clearer, though either way should be effectively the same.
I'll move it to the xaml if that's what you prefer.

> Also did you check this on 2 monitors with different DPI? I am afraid it might not work correctly

I tested it on my secondary monitor at both 150% and 175% scaling, and it appears to function normally. I did notice some odd behavior when quickly zooming on the larger monitor and then zooming again on the smaller monitor, (zoom window appears far below where it should) but when I tried it on the live build (v0.37.0) the exact same thing happened, so I think it's safe to say my code hasn't caused any *new* issues there. More testing would be appreciated though.

Reply:
how was this maxed before?

Reply:
> how was this maxed before?
Previously, the window width/height matched the width/height of its' contents, so WPF just calculated the correct size.
Unfortunately, this also meant changing the window's width/height continuously throughout an animation, and by extension the top-left of the window had to be animated to keep the window centered.
Keeping the window size constant allows us to set the window position *once*, then animate freely within the window, which results in a much smoother animation.

Reply:
The odd behavior I noticed when quickly moving the zoom window between monitors with different DPI seems to happen **much** more often with these changes than on v0.37.0, so I think a fix should also be included in this pull request.